db/migrate/20160425161234_remove_deleted_tables_from_replication_settings.rb

```diff
@@ -0,0 +1,16 @@
+class RemoveDeletedTablesFromReplicationSettings < ActiveRecord::Migration[5.0]
+  class SettingsChange < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+    serialize :value
+  end
+
+  def up
+    deleted_tables = %w(miq_events miq_license_contents vim_performances)
```

Comment:
Table was renamed in that PR, but not in the configurations table.  On upgrade a user won't have miq_event_definitions, but will have miq_events.  The migration needs to remove one and add the other.